console/src/main/scripts/plugins/metrics/ts/app-details/appServerDatasourcesDetails.ts

```diff
@@ -240,6 +224,27 @@ module HawkularMetrics {
       this.getDrivers();
     }
 
+    private getAlerts(res:any):void {
+      let dsArray:IAlert[];
+      let promise = this.HawkularAlertsManager.queryAlerts({
+        statuses: 'OPEN',
+        tags: 'resourceId|' + res.id,
+        startTime: this.startTimeStamp,
+        endTime: this.endTimeStamp
+      }).then((data)=> {
```

Comment:
The type would be good here too as anyone outside of alerts wouldn't know what kind of data is in 'data'. The types let our IDE's autocomplete this stuff.